src/libos/src/fs/syscalls.rs

```diff
@@ -35,6 +37,54 @@ pub fn do_eventfd2(init_val: u32, flags: i32) -> Result<isize> {
     Ok(fd as isize)
 }
 
+pub fn do_timerfd_create(clockid: u32, flags: i32) -> Result<isize> {
+    info!("timerfd: clockid {}, flags {} ", clockid, flags);
```

Comment:
`info!` is used to show some critical information(e.g. process creation), please change to `debug!`
```
debug!("timerfd_create: clockid {}, flags {} ", clockid, flags);
```

Reply:
Done